lib/datadog/tracing/distributed/baggage.rb

```diff
@@ -0,0 +1,133 @@
+# frozen_string_literal: true
+
+require_relative '../metadata/ext'
+require_relative '../trace_digest'
+require_relative 'datadog_tags_codec'
+require_relative '../utils'
+require_relative 'helpers'
+require 'uri'
+require 'cgi'
```

Comment:
```suggestion
require 'cgi'
```
This is no longer used.